EducationalCenter/EducationalCenter.Angular/Controllers/CoursesController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using EducationalCenter.BLL.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace EducationalCenter.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CoursesController : ControllerBase
+    {
+        private ICourseService _courseService;
+
+        public CoursesController(ICourseService courseService)
+        {
+            _courseService = courseService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetAll()
+        {
+            var courses = await _courseService.GetAllAsync();
```

Comment:
Use mapping to CourseDTO would be better here

Reply:
Why? It doesn`t feel right for me when businesslogic layer (BLL) returns a model from dataaccess layer (DAL) instead of DTO model. I supposed that BLL should protect models from DAL and controller should know as less as possible about DAL.